src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -3,6 +3,7 @@
 use DateTime;
 use Exception;
 use ArrayAccess;
+use browner12\money\Money;
```

Comment:
good to know. though it was semi-alphabetical, but now i see how stupid that was